README.md

```diff
@@ -63,7 +68,37 @@ This file should declare those resources, and any
 customization to apply to them, e.g. _add a common
 label_.
 
-![base image][imageBase]
+```
+
+base: kustomization + resources
+
+kustomization.yaml                      deployment.yaml                                                 service.yaml
++-------------------------------+       +-------------------------------------------------------+       +-----------------------------------+
+| commonLabels:                 |       | apiVersion: apps/v1                                   |       | apiVersion: v1                    |
+|   app: myapp                  |       | kind: Deployment                                      |       | kind: Service                     |
+| resources:                    |       | metadata:                                             |       | metadata:                         |
+|   - deployment.yaml           |       |   name: myapp                                         |       |   name: myapp                     |
+|   - service.yaml              |       | spec:                                                 |       | spec:                             |
+| configMapGenerator:           |       |   selector:                                           |       |   selector:                       |
+|   - name: myapp-map           |       |     matchLabels:                                      |       |     app: myapp                    |
+|     files:                    |       |       app: myapp                                      |       |   ports:                          |
+|       - env.startup.txt       |       |   template:                                           |       |     - port: 6060                  |
```

Comment:
I just realized that this env file isn't really in the sample (and never was). Let's change the generator to use literals to make the example work while keeping it simple.

```
configMapGenerator:
  - name: myapp-map
    literals:
      - KEY=value
```

I also just noticed that this example kustomization and the one below are missing the type information. This currently works, but including it is recommended, so might as well have it in the example:

```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
```